approval_polls/templates/approval_polls/edit.html

```diff
@@ -90,6 +91,12 @@ <h5 style="color:#8B0000;"><b>Please ensure you've entered all choices correctly
         </div>
       {% endfor %}
       {% endwith %}
+      <div id='tag-input'>
+        <h6>(Enter a comma after each tag:)</h6>
```

Comment:
Something very minor but I think it would be nice if you could remove the colon here and in the prompt for email addresses. I was concerned as I have rarely seen both used in the same context.